doc/rs/crate/moq-lite-api.md

```
@@ -0,0 +1,471 @@
+    
+    Ok(())
+}
+```
+
+---
+
+## Core Types
+
+### Hierarchy
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)